docs/src/pages/[platform]/connected-components/authenticator/headless/headless-usage.web.mdx

```diff
@@ -56,7 +56,7 @@ Here's an example that uses the `toResetPassword` trigger transition, to create
         <TableRow>
           <ResponsiveTableCell label="Name">`user`</ResponsiveTableCell>
           <ResponsiveTableCell label="Description">Current signed in user</ResponsiveTableCell>
-          <ResponsiveTableCell label="Type">[`CognitoUserAmplify`](https://github.com/aws-amplify/amplify-ui/blob/f9be291e4b89137d7b521bd3e82b4f96ee4323f6/packages/ui/src/types/authMachine.ts#L113-L115)</ResponsiveTableCell>
```

Comment:
Removed this link as it points to a commit, in a file that no longer exists in `main`